src/borg/remote.py

```diff
@@ -513,6 +513,7 @@ def __init__(self, location, create=False, exclusive=False, lock_wait=None, lock
         self.chunkid_to_msgids = {}
         self.ignore_responses = set()
         self.responses = {}
+        self.async_responses = {}
```

Comment:
yes, currently that would be more specific.
but maybe we want later also non-error responses there? so i just used the more general term.
in fact i already had that, when I noticed they do not return anything anyway.